apps/site/src/lib/decisionScores.ts

```diff
@@ -0,0 +1,18 @@
+import { Decision } from "./userRecord";
+
+/** For use in nonhacker applicant reviews.*/
+const acceptScore = "100";
+const waitlistScore = "-1";
+const rejectScore = "0";
```

Comment:
Question: is there a specific reason these are strings rather than numbers?

Reply:
The Input component for inputting scores in `HackerApplicantActions.tsx` returns a string when calling `onChange`, so the `submitReview` function in `useApplicant.ts` expects a string.

Because I'm using the same route for submitting hacker, mentor, and volunteer reviews, I made these specific scores also strings.

Something I could do is set the score parameter for `submitReview` to be of type `string | number` so that these scores can be numbers.

Reply:
I personally think rather than using type `string | number` it'd be better to leave these score types as `number` types and then converting the `value` argument string into a `number` before passing it into `submitReview` in `HackerApplicantActions`